cli/template/extras/src/trpc/react.tsx

```diff
@@ -15,15 +15,20 @@ export function TRPCReactProvider(props: { children: React.ReactNode }) {
 
   const [trpcClient] = useState(() =>
     api.createClient({
-      transformer,
+      transformer: SuperJSON,
       links: [
         loggerLink({
           enabled: (op) =>
             process.env.NODE_ENV === "development" ||
             (op.direction === "down" && op.result instanceof Error),
         }),
         unstable_httpBatchStreamLink({
-          url: getUrl(),
+          url: getBaseUrl() + "/api/trpc",
+          async headers() {
```

Comment:
```suggestion
          transformer: SuperJSON,
          async headers() {
```

Apply [Transformers are moved to links (breaking)](https://trpc.io/docs/migrate-from-v10-to-v11#transformers-are-moved-to-links-breaking) migration